sdks/java/harness/src/test/java/org/apache/beam/fn/harness/FnApiDoFnRunnerTest.java

```diff
@@ -1604,6 +1606,11 @@ public ProcessContinuation processElement(
         }
       }
 
+      @Override
+      public Duration getAllowedTimestampSkew() {
```

Comment:
Why this change?

Reply:
This test was triggering the check. I believe because it outputs timestamps at a given position past the min timestamp which seemed sufficiently weird / hacky that I'm not surprised it was triggering it.